### ISSUE TITLE
Issue running LabelImage.java demo. 

### ISSUE THREAD

**Issue by louisquinn** [tensorflow/tensorflow#8378]
Below is the error:

Exception in thread "main" java.lang.UnsupportedOperationException: Op BatchNormWithGlobalNormalization is not available in GraphDef version 21. It has been removed in version 9. Use tf.nn.batch_normalization().
	at org.tensorflow.Graph.importGraphDef(Native Method)
	at org.tensorflow.Graph.importGraphDef(Graph.java:113)
	at org.tensorflow.Graph.importGraphDef(Graph.java:97)
	at org.tensorflow.examples.LabelImage.executeInceptionGraph(LabelImage.java:110)
	at org.tensorflow.examples.LabelImage.main(LabelImage.java:65)

I can't find the source file in which to use tf.nn.batch_normalization
Are the Java libraries using older versions of Tensorflow?

**Comment by asimshankar** [comment]
Could you provide some more detail (ideally the set of commands used to reproduce the problem)? Following the instructions in [java/README.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/java/README.md), I was unable to reproduce the error. Are you using a custom graph perhaps?

For example, the following does not reproduce the problem:
```sh
# Download and unzip the model
wget "https://storage.googleapis.com/download.tensorflow.org/models/inception5h.zip"
unzip inception5h.zip

# Download a sample image
wget "https://github.com/tensorflow/tensorflow/raw/master/tensorflow/examples/label_image/data/grace_hopper.jpg"

# Run the example as per instructions in README.md
java \
  -Djava.library.path=./jni \
  -cp libtensorflow-1.0.0-PREVIEW1.jar:./src/main/java \
  org.tensorflow.examples.LabelImage . ./grace_hopper.jpg
```

**Comment by kinDSa** [comment]
I get same error while train own model and [followed codelab ](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets/#0) instruction.

**Comment by asimshankar** [comment]
Ah, thanks for the update @kinDSa 
I think this is because the codelab is using a very old graph with deprecated operations. I'm going to close this as a duplicate of #8396 (since that has more information) and follow up there.

**Comment by louisquinn** [comment]
Ahhh, downloading that version of inception fixed it. 
I was using the version from here:
https://storage.googleapis.com/download.tensorflow.org/models/inception_dec_2015.zip

from the image recognition tutorial:
https://www.tensorflow.org/tutorials/image_recognition

What is the difference between the two versions? It seems the one I downloaded is more recent. 
I'm using the bazel command in the readme
